extra/exe-builder/Program.cs

```diff
@@ -204,8 +219,12 @@ public class UptimeKumaApplicationContext : ApplicationContext
 
         }
 
-        void VisitGitHub(object sender, EventArgs e)
-        {
+        void VisitGitHub(object sender, EventArgs e) {
+            if (intranetOnly) {
+                MessageBox.Show("You have parsed in --intranet so we will not try to access the internet or visit github.com, please go to https://github.com/louislam/uptime-kuma if you want to visit github.");
```

Comment:
I just noticed that you used `MessageBox.Show` instead of `Console.WriteLine` in these cases.

Is `CheckForUpdate` and `DownloadFiles` something that is triggered by a user?
If yes, the  `MessageBox.Show` was actually a good idea ^^

Reply:
When you click right mouse on the uptime kuma icon you get these options.
I wasn't sure if it was against the guidelines, I'll make sure to add it back!